modules/jdbc/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnection.java

```diff
@@ -78,6 +79,8 @@ public class JdbcConnection implements Connection {
     /** Statements modification mutex. */
     private final Object stmtsMux = new Object();
 
+    private final AtomicLong tokenGenerator = new AtomicLong(Long.MIN_VALUE);
```

Comment:
Are you afraid it will be insufficient if we start from 0?

Reply:
this, actually, artifact of debugging... Correlation token is of primitive type in all the request objects, therefore when you forget to amend serialization, request won't fail, but will have incorrect attributes instead. Setting initial value different from zero helped me to find all such missed places.

Anyway, removed initialization with custom value to avoid confusion (btw, AtomicLong do wrap around on boundaries, so we have here effectively infinite amount of tokens)